docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -105,6 +105,7 @@ To add a custom Git repository that contains your Helm charts or cluster templat
 1. Click **Create**.
 1. Select the target, **Git repository containing Helm chart...**.
 1. You must enter a name and a Git repository URL. The other fields, including the description, are optional. Enter an alternative branch name if you don't want to pull from whichever branch the repo owner has set as the default. Usually, the default branch is named either `main` or `master`.
+1. Input **Refresh Interval** seconds value. This will tell Rancher manager at what interval the Helm repository must be refreshed. The default is 21600 seconds.
```

Comment:
```suggestion
1. You must enter a name and a Git repository URL. The other fields, including the description, are optional. 
  - Setting the alternative branch name allows you to pull from a branch other than the branch the repo owner has set as the default. Usually, the default branch is named either `main` or `master`.
  - Setting the **Refresh Interval** alters how many seconds Rancher waits before refreshing the Helm repository. The default value is 21600 seconds (24 hours).
```

Reply:
Is refresh interval optional? I revised on the assumption that it is, since the text describes a default value.